controllers/export_controller.go

```diff
@@ -47,12 +47,12 @@ import (
 	"sigs.k8s.io/controller-runtime/pkg/controller/controllerutil"
 	ctrllog "sigs.k8s.io/controller-runtime/pkg/log"
 
-	primerv1alpha1 "github.com/cooktheryan/gitops-primer/api/v1alpha1"
+	primerv1alpha1 "github.com/kubesaw/snapshot-operator/api/v1alpha1"
 )
 
 const (
-	gitopsPrimerLabel = "openshift.gitops.primer"
-	finalizer         = "openshift.gitops.primer"
+	snapshotOperatorLabel = "openshift.snapshot.operator"
```

Comment:
I would leave out changing the code at the moment. But in any case, I'm not sure this label conforms to the other labels we use in codeready-toolchain-soon-to-be-kubesaw. Something like `snapshot.dev.openshift.com/export-of=...` would imho be more appropriate.

Reply:
I've changed this to your suggestion.